client/landing/gutenboarding/gutenboard.tsx

```diff
@@ -62,7 +74,12 @@ export function Gutenboard() {
 								aria-label={ NO__( 'Onboarding screen content' ) }
 								tabIndex={ -1 }
 							>
-								<BlockList className="gutenboarding-block-list" />
+								<BlockList
+									__experimentalUIParts={ {
+										hasPopover: false,
+										hasSelectedUI: false,
+									} }
```

Comment:
The smaller and more focused we can keep PRs, the better. This would probably have made a nice, small, contained PR by itself. This PR has a lot going on and adding more tends to slow things down.

No problem here, just something to keep in mind 👍

Reply:
It did slow things down! Lesson learned!